auth_otp/management/commands/cleanup_otp.py

```
@@ -0,0 +1,102 @@
+    
+    def _cleanup_otps(self, dry_run, age_hours):
+        """پاکسازی OTP های قدیمی"""
+        if dry_run:
+            from auth_otp.models import OTPRequest
+            cutoff = timezone.now() - timedelta(hours=age_hours)
+            count = OTPRequest.objects.filter(created_at__lt=cutoff).count()
+            self.stdout.write(
+                f'تعداد OTP های قابل حذف (قدیمی‌تر از {age_hours} ساعت): {count}'
+            )
+            return count
+        else:
```

Ruff (0.12.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)